### DISCUSSION TITLE
Tree: horizontal scroll

### DISCUSSION THREAD

**Discussion by saskliutas** [iTwin/iTwinUI#2146] [General]
Is it possible to add horizontal scroll to the tree component in 3.13.0? We usually encounter quite deep tree structures with long labels. Before 3.13.0 it was possible to add horizontal scroll doing something like this: https://stackblitz.com/edit/github-mmlry8-kslxf8?file=src%2FApp.tsx

But I didn't manage to achieve the same in 3.13.0.

**Comment by Ben-Pusey-Bentley** [comment]
Hey, thanks for posting a discussion on this. After some investigation, it seems to be an issue being caused by our recent refactor of the tree component's virtualization. We are currently investigating potential solutions to this issue.
